webplugin/js/app/labels/default-labels.js

```diff
@@ -104,6 +104,7 @@ class KMLabel {
                 'Uh oh! No internet connection. Please check your network settings and try again.',
             'socket-disconnect.msg':
                 'Error while syncing messages. Check your firewall settings or try again after some time.',
+            'business-hour.msg': 'You have reached us outside of our business hours. Our Bot will handle your queries till the team is back online',
```

Comment:
You have reached us outside our business hours. Our bot will handle your queries until the team is back online.

Reply:
done